<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>My App</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>391</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLabel" name="Naslov">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>331</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Dobrodošli u Qt aplikaciju!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="OpisApp">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>311</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Molimo unesite neki tekst (do 200 znakova) i pritisnite &quot;OK&quot; gumb za nastavak. Za brisanje unesenog teksta pritisnite &quot;Obriši&quot;.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="InputLine">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>SizeVerCursor</cursorShape>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="OKgumb">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>370</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ObrisiGumb">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Obriši</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>471</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <widget class="QMenu" name="menuJezik">
    <property name="geometry">
     <rect>
      <x>339</x>
      <y>116</y>
      <width>159</width>
      <height>126</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
     </font>
    </property>
    <property name="title">
     <string>Jezik</string>
    </property>
    <addaction name="actionHrvatski"/>
    <addaction name="actionEngleski"/>
    <addaction name="actionJapanski"/>
   </widget>
   <widget class="QMenu" name="menuParametri">
    <property name="geometry">
     <rect>
      <x>386</x>
      <y>116</y>
      <width>159</width>
      <height>102</height>
     </rect>
    </property>
    <property name="title">
     <string>Parametri</string>
    </property>
    <addaction name="actionDefault"/>
    <addaction name="actionCustom"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionInfo"/>
   </widget>
   <addaction name="menuJezik"/>
   <addaction name="menuParametri"/>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionHrvatski">
   <property name="text">
    <string>Hrvatski</string>
   </property>
  </action>
  <action name="actionEngleski">
   <property name="text">
    <string>Engleski</string>
   </property>
  </action>
  <action name="actionJapanski">
   <property name="text">
    <string>Japanski</string>
   </property>
  </action>
  <action name="actionDefault">
   <property name="text">
    <string>Zadani</string>
   </property>
  </action>
  <action name="actionCustom">
   <property name="text">
    <string>Prilagođeni...</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>O aplikaciji</string>
   </property>
  </action>
  <action name="actionO_Qt_platformi">
   <property name="text">
    <string>O Qt platformi</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
